Makefile

```diff
@@ -342,6 +342,11 @@ ifeq ($(ATFTPD),)
 ATFTPD:=/usr/sbin/atftpd
 endif
 
+# Requires HPA in.tftpd not traditional BSD in.tftpd
+# Unfortunately in.tftpd seems to require root always
+# even if run as current user, otherwise it reports
+# "cannot set groups for user $USER"
+#
 IN_TFTPD:=$(shell which in.tfptd)
```

Comment:
Here `tftpd` is spelled wrong

Reply:
I also think that `in.tftpd` is not normally in non-root users path.

Reply:
@stffrdhrn Pushed new patch (finally!) fixing the typo in `$(which in.tftpd)`, which I thought I'd fixed previously, but apparently never in a branch I pushed :-(   (You've pointed it out several times!)

@mithro `in.tftpd` is in my path :-)  But yes, I'm a bit unusual.  There's a hard coded default assumption in case the `$(which in.tftpd)` wasn't matched, which is what was working before.  I think it's worth trying the `$(which ...)` in case someone has installed it as a local file, rather than a system package.